src/main/java/gameframework/assets/Sound.java

```diff
@@ -64,6 +64,72 @@ public Sound(final String path) throws UnsupportedAudioFileException, IOExceptio
 	}
 
 	/**
+	 * Creates and loads the sound with a different speed
```

Comment:
This constructor duplicates most of the previous one. Please factor out common code